### ISSUE TITLE
Unexpected behaviours after a screen orientation change (reported by activity)

### ISSUE THREAD

**Issue by dlicse1** [Freeyourgadget/Gadgetbridge#1589]
I developed an automatic test case generation tool and it detected these problems in your app.
I would like to know if you consider them as failures and, if not, why. Thanks for your collaboration.

1) SettingsActivity, when I change the orientation of the device screen, changes its state. Below there are two different examples: the screenshots were taken before and after the screen orientation change and they are paired (before, after), (before, after) 

**--- first case ---**

"Settings" becomes "Ajustes".

![2019-07-11 19-36-47_before](https://user-images.githubusercontent.com/53469180/62141021-356ef180-b2ec-11e9-8598-52634a13165e.png)
![2019-07-11 19-36-47_after](https://user-images.githubusercontent.com/53469180/62141023-36078800-b2ec-11e9-997c-f816b98ce96b.png)

**--- second case ---**

From "Right To Left Support" it returns back to the previous options.

![2019-07-11 19-29-00_before](https://user-images.githubusercontent.com/53469180/62141024-36078800-b2ec-11e9-8050-aa2cecbc5d5f.png)
![2019-07-11 19-29-00_after](https://user-images.githubusercontent.com/53469180/62141025-36078800-b2ec-11e9-8f9c-a193cb4d57b9.png)

When I change the orientation of the device screen, ControlCenterv2 crashes
![2019-07-11 18-37-19_before](https://user-images.githubusercontent.com/53469180/62140884-02c4f900-b2ec-11e9-8b89-2b6913a53491.png)
![2019-07-11 18-37-19_after](https://user-images.githubusercontent.com/53469180/62140886-02c4f900-b2ec-11e9-9291-a3fc1ad00e8e.png)

3) AppBlacklistActivity, when I change the orientation of the device screen, changes its state. Below there are two different examples: the screenshots were taken before and after the screen orientation change and they are paired (before, after), (before, after)

**--- first case ---**

The "CUT, PASTE, ..." options disappear.

![2019-07-12 01-00-06_before](https://user-images.githubusercontent.com/53469180/62140681-b8437c80-b2eb-11e9-8367-85ae006bb1f9.png)
![2019-07-12 01-00-06_after](https://user-images.githubusercontent.com/53469180/62140683-b8437c80-b2eb-11e9-8d00-04096ae4db54.png)

**--- second case ---**

The blacklist gets shuffled

![2019-07-11 19-18-29_before](https://user-images.githubusercontent.com/53469180/62140687-b8dc1300-b2eb-11e9-896c-912f07afed58.png)
![2019-07-11 19-18-29_after](https://user-images.githubusercontent.com/53469180/62140688-b8dc1300-b2eb-11e9-87a8-e3ea70ed1c45.png)

**Comment by vanous** [comment]
Yes, thank you. Basically a duplicate/version of https://github.com/Freeyourgadget/Gadgetbridge/issues/1429

**Comment by dlicse1** [comment]
Thank you very much!
